platform-api-docs/scripts/specs/orgs-workspaces-operations-overlay-1.88.yml

```diff
@@ -0,0 +1,292 @@
+  
+  # ---- LIST WORKSPACE PARTICIPANTS ----
+  
+  - target: "$.paths./orgs/{orgId}/workspaces/{workspaceId}/participants.get.summary"
+    update: "List workspace participants"
+  
+  - target: "$.paths./orgs/{orgId}/workspaces/{workspaceId}/participants.get.description"
+    update: "Lists all participants of the workspace identified by the given `workspaceId`. Participants are users or teams who have been granted access to a workspace and can be assigned roles such as owner, admin, maintain, or view."
+  
+  # ---- CREATE WORKSPACE PARTICIPANT ----
+  
+  - target: "$.paths./orgs/{orgId}/workspaces/{workspaceId}/participants/add.put.summary"
+    update: "Create workspace participant"
+  
+  - target: "$.paths./orgs/{orgId}/workspaces/{workspaceId}/participants/add.put.description"
+    update: "Adds a new participant to the workspace identified by the given `workspaceId`. The participant can be either an individual user or a team, and is assigned the specified workspace role."
+  
+  # ---- DELETE WORKSPACE PARTICIPANT ----
+  
+  - target: "$.paths./orgs/{orgId}/workspaces/{workspaceId}/participants/{participantId}.delete.summary"
+    update: "Delete workspace participant"
+  
+  - target: "$.paths./orgs/{orgId}/workspaces/{workspaceId}/participants/{participantId}.delete.description"
+    update: "Removes the participant identified by the given `participantId` from the given workspace."
+  
+  # ---- UPDATE WORKSPACE PARTICIPANT ROLE ----
+  
+  - target: "$.paths./orgs/{orgId}/workspaces/{workspaceId}/participants/{participantId}/role.put.summary"
+    update: "Update workspace participant role"
+  
+  - target: "$.paths./orgs/{orgId}/workspaces/{workspaceId}/participants/{participantId}/role.put.description"
+    update: "Updates the role of the participant identified by the given `participantId` in the given workspace. Available roles are owner, admin, maintain, view, and launch."
+  
+  # ---- DESCRIBE WORKSPACE STUDIOS SETTINGS ----
+  
+  - target: "$.paths./orgs/{orgId}/workspaces/{workspaceId}/settings/studios.get.summary"
+    update: "Describe workspace Studios settings"
+  
+  - target: "$.paths./orgs/{orgId}/workspaces/{workspaceId}/settings/studios.get.description"
+    update: "Retrieves the Studios configuration settings for the workspace identified by the given `workspaceId`. Studios settings control default values for Studios instances created in this workspace."
+  
+  # ===== USER WORKSPACES - OPERATIONS =====
+  
+  # ---- LIST USER WORKSPACES ----
+  
+  - target: "$.paths./user/{userId}/workspaces.get.summary"
+    update: "List user workspaces"
+  
+  - target: "$.paths./user/{userId}/workspaces.get.description"
+    update: "Lists all workspaces and organizations to which the user identified by the given `userId` belongs. This includes both workspaces where the user is a direct participant and organizations where the user is a member."
```

Comment:
> This includes both workspaces where the user is a direct participant

I would say just `where the user is a participant`. We tend to use `direct participant` when the user is assigned as a participant to the workspace directly, but the user can also have access to the workspace through a team that is assigned as a participant to the workspace.